content/en/continuous_integration/setup_tests/python.md

```diff
@@ -11,25 +11,17 @@ further_reading:
 ---
 ## Compatibility
 
-Supported Test Frameworks:
-* Pytest 3.0+
-
-Supported CI providers:
-* Appveyor
-* Azure Pipelines
-* BitBucket
-* BuildKite
-* CircleCI
-* GitHub Actions
-* GitLab
-* Jenkins
-* TravisCI
+Supported Python interpreters:
+* Python >= 2.7 and >= 3.5
+
+Supported test frameworks:
+* pytest >= 4.0.0
```

Comment:
Our pytest integration is compatible with:
- Python 2: pytest<=4.6 (4.6 is last to support python 2)
- Python 3: pytest >= 3.0.0